docs/pictures/khoa/AddProfile.puml

```diff
@@ -1,6 +1,5 @@
 @startuml
-!define PLANTUML_HOME %dirpath()/../../umldiagram
-!include PLANTUML_HOME/style.puml
```

Comment:
Perhaps you can help me keep this line so that the diagrams have consistent format?